### DISCUSSION TITLE
THANK YOU

### DISCUSSION THREAD

**Discussion by nitsujri** [linearmouse/linearmouse#21] [General]
I just wanted to say thank you so much for this amazing repo and app. This seemingly simple request is insanely hard to pull off well in OSX.

**Comment by lujjjh** [comment]
You are welcome! I'm also willing to see more people join in this project and more FOSS on macOS.

(As this is not an issue, I'll convert it to discussion.)

**Comment by longshorej** [comment]
Ditto - I've been looking for a solution to this for years. This utility has been perfect. Thanks!

**Comment by nitsujri** [comment]
I posted it on StackExchange, please upvote!

https://apple.stackexchange.com/questions/333989/how-do-i-disable-mouse-acceleration-and-adjust-tracking-speed-in-high-sierra/431086#431086

**Comment by giuliojiang** [comment]
Absolutely, top quality app. Thank you so much!

**Comment by JonasBoi** [comment]
Thank you so much, been looking for something that is simple and 'just works' for a while. Finally found it.

**Comment by tea-lover-418** [comment]
This tool absolutely shouldn't have to exist, people at apple are a bunch of clowns.

THANKS!
